test/tools/IsolTestOptions.cpp

```diff
@@ -59,11 +59,20 @@ std::string editorPath()
 IsolTestOptions::IsolTestOptions(std::string* _editor):
 	CommonOptions(description)
 {
+	editor = _editor;
+	enforceViaYul = true;
+	enforceGasTest = (evmVersion() == langutil::EVMVersion{});
+	enforceGasTestMinValue = 100000;
```

Comment:
Since this is a constructor, you should put these on the constructor initializer list rather than in the body.

Putting them directly on the struct fields would also be fine - that's how `CommonOptions` already does it after all.

Reply:
I can and will move the `editor` assignment to an initializer list, but the rest are members of the `CommonOptions` base class so, as far as I know, that is not valid syntax to initialize them on a list from the derived class. I did it this way to avoid having to pass a parameter for every single option to the base class constructor to modify defaults via the derived class; they can just be assigned in the derived class constructor after initialization by the base class and keep the interface simpler.

Reply:
Ah, right. Forgot that they're from the base class. It's fine then.